mc2/controllers/base/models.py

```diff
@@ -155,7 +158,9 @@ def get_marathon_app_data(self):
             "instances": self.marathon_instances,
         }
 
-        if self.marathon_cmd:
+        if self.marathon_args:
+            data.update({"args": shlex.split(self.marathon_args)})
+        elif self.marathon_cmd:
```

Comment:
Not sure if we want to grey-out/disable the cmd form if the args form is used or something?

Reply:
I think the easiest for now is to add validation that will raise an error when both are specified. Would you be happy with that as a compromise? @JayH5

Reply:
Sounds good. I'm pretty shaky on the Django stuff so will take others' word for whatever is best.

Reply:
Something like this
![mission_control_args_validation](https://user-images.githubusercontent.com/623417/37141372-73064586-22bd-11e8-9bfb-b4f61ff0c386.jpg)

Reply:
Thanks, looks good 👍 🍰